.storybook/addons/markdown.js

```diff
@@ -0,0 +1,70 @@
+import addons, { types, makeDecorator } from '@storybook/addons';
+import React, { Component } from 'react';
+
+const ADDON_ID = 'markdown';
+const ADDON_EVENT = `${ADDON_ID}/event`;
+const PANEL_ID = `${ADDON_ID}/panel`;
+const DEFAULT_MSG = 'No Markdown Provided';
+
+const uuidv4 = () => {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
+    var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
+    return v.toString(16);
+  });
+};
+
+const e = React.createElement;
+
+addons.register(ADDON_ID, (api) => {
+  const title = 'Readme';
+  const channel = addons.getChannel();
+  let initalText = DEFAULT_MSG;
+
+  class Wrapper extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        html: initalText,
+      };
+      this.onAddonAdded = this.onAddonAdded.bind(this);
+    }
+
+    onAddonAdded ({ html }) {
+      initalText = html;
```

Comment:
Typo initial and why the overwrite here? not strong opinions here

Reply:
@MKaHead thanks, cause of multiple re-rendering we need to store last one outside react. Is like setting default dynamically. Will fix typo